python/README.md

```diff
@@ -0,0 +1,58 @@
+# DotSlash: simplified executable deployment
+
+[![CI - Test](https://github.com/facebook/dotslash/actions/workflows/test-python.yml/badge.svg)](https://github.com/facebook/dotslash/actions/workflows/test-python.yml)
+[![PyPI - Version](https://img.shields.io/pypi/v/dotslash.svg?logo=pypi&label=PyPI&logoColor=gold)](https://pypi.org/project/dotslash/)
+[![PyPI - Downloads](https://img.shields.io/pypi/dm/dotslash.svg?color=blue&label=Downloads&logo=pypi&logoColor=gold)](https://pypi.org/project/dotslash/)
+[![Built by Hatch](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/pypa/hatch/master/docs/assets/badge/v0.json)](https://github.com/pypa/hatch)
+[![Ruff linting/formatting](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
```

Comment:
This isn't right anymore, is it?  I guess technically we kinda do use `ruff` via `ufmt`, so maybe it sorta is?  I won't block on this; we can fix it post merge.

Reply:
I don't think the badge requires the direct use of the CLI, as long as the tool is used.